tests/python/unittest/test_numpy_op.py

```diff
@@ -554,6 +554,65 @@ def hybrid_forward(self, F, x):
                 assert same(mx_out.asnumpy(), np_out)
 
 
+@with_seed()
+@npx.use_np_shape
+def test_np_linspace():
+    configs = [
+        (0.0, 1.0, 10),
+        (-2, 4, 45),
+        (5.234324, 8.98324, 324)
```

Comment:
Does these 3 cases provide full coverage of all cases? Maybe we can take a look at numpy's test for linspace as a reference?

Reply:
checked the numpy unit test, and added more here